docs/advanced-topics/networkobject-parenting.md

```diff
@@ -60,27 +45,52 @@ If you run into a situation where your client must trigger parenting a `NetworkO
 
 ### Invalid `NetworkObject` Parenting Will Be Reverted
-If an invalid/unsupported `NetworkObject` parenting happens, Netcode will immediately revert it back to its original parenting status. 
+If an invalid/unsupported `NetworkObject` parenting happens, the attempted parenting action will be immediately reverted it back to its original parenting status. 
 
 **For example:** 
 If you had a `NetworkObject` who's current parent was root and tried to parent it in an invalid way (i.e. under a GameObject without a `NetworkObject` component) then a warning message would be logged and the `NetworkObject` would revert back to having root as its parent.
 
 ### In-scene NetworkObject parenting of players
-In-scene placed `NetworkObject` parenting of players requires the client to be synchronized first.  Since a server can only perform parenting related actions, the server must have already received the `NetworkSceneManager` generated `SceneEventType.SynchronizeComplete` message before the server can parent the client's player `NetworkObject`.
+In-scene placed `NetworkObject` parenting of players requires the client to be synchronized first.  Since a server can only perform parenting related actions, the server must have already received the `NetworkSceneManager` generated `SceneEventType.SynchronizeComplete` message  before the server can parent the client's player `NetworkObject`.
 :::info
 For more information, see the "[Real World In-scene NetworkObject Parenting of Players Solution](inscene_parenting_player.md)".
 :::
 
+### WorldPositionStays usage
+When using the `NetworkObject.TrySetParent` or `NetworkObject.TryRemoveParent` methods, the `WorldPositionStays` parameter is synchronized with currently connected and late joining clients. When removing a child from its parent, you should use the same `WorldPositionStays` value that was used to parent the child.  More specifically when `WorldPositionStays` is set to false this applies, but if you are using the defalt value of `true` then this is not required (because it is the default).
+
+When the `WorldPositionStays` parameter in `NetworkObject.TrySetParent` is the default value of `true`, this will preserve the world space values of the child `NetworkObject` relative to the parent.  However, sometimes you might want to only preserve the local space values (i.e. pick up an object that only has some initial changes to the child's transform when parented).  Through a combination of `NetworkObject.TrySetParent` and `NetworkBehaviour.OnNetworkObjectParentChanged` you can accomplish this without the need for a `NetworkTransform`.  To better understand how this works, it is important to understand the order of operations for both of these two methods:
```

Comment:
> When the `WorldPositionStays` parameter in `NetworkObject.TrySetParent` is the default value of `true`, NGO preserves the world space values of the child `NetworkObject` relative to the parent. However, sometimes you might want only to preserve the local space values (that is, pick up an object that only has some initial changes to the child's transform when parented). You can accomplish this without needing a `NetworkTransform` through a combination of `NetworkObject.TrySetParent` and `NetworkBehaviour.OnNetworkObjectParentChanged`. To better understand how this works, it is important to understand the order of operations for both of these two methods:

Reply:
We are allowed to use NGO?

Reply:
That's a good question - one I've been wondering, too. In the gaming service documentation (e.g., Relay), we refer to Netcode for GameObjects as NGO (after introducing the acronym). Maybe that's not how we want to refer to it, though. Especially since we called it "netcode" everywhere else, this confused me a bit when I first started working with netcode.

Do you know who might be able to say authoritatively what we should and shouldn't use to refer to netcode?